src/main/java/org/amahi/anywhere/adapter/RecentFilesAdapter.java

```diff
@@ -60,7 +61,7 @@ public void onBindViewHolder(RecentFilesViewHolder holder, int position) {
 
     private void setUpViewHolder(RecentFile file, RecentFilesViewHolder fileHolder) {
         Uri uri = Uri.parse(file.getUri());
-        String name = uri.getQueryParameter("p").substring(uri.getQueryParameter("p").lastIndexOf('/')+1);
+        String name = uri.getQueryParameter("p").substring(uri.getQueryParameter("p").lastIndexOf('/') + 1);
```

Comment:
Did you reformatted the whole file? Make sure you only reformat the VCS changed text. Reformatting files that are needed can be done in a separate PR. These little tips will help you in the future.

Reply:
Thanks for the suggestion.